test/auto_parallel/hybrid_strategy/parallel_api.py

```diff
@@ -190,94 +232,137 @@ def check_mp(self, layer, share_embedding):
+    def parallel_model(self, layer):
         dp_config = None
         mp_config = None
         pp_config = None
+        prefix = "model." if self.test_lora else ""
         if self.pp > 1:
             # decoders_per_rank = self.config.num_hidden_layers // self.pp
             # split_spec = {
-            #     f"llama.layers.{i * decoders_per_rank - 1}": SplitPoint.END
+            #     ff"{prefix}llama.layers.{i * decoders_per_rank - 1}": SplitPoint.END
             #     for i in range(1, self.pp)
             # }
             pp_config = {
-                'split_spec': "llama.layers",
-                "global_spec": "llama.global_layer",
+                'split_spec': f"{prefix}llama.layers",
+                "global_spec": f"{prefix}llama.global_layer",
             }
         if self.dp > 1:
             dp_config = {'sharding_level': self.level}
         if self.mp > 1:
             if not self.sequence_parallel:
                 plan = {
-                    "llama.embed_tokens": dist.ColWiseParallel(
+                    f"{prefix}llama.embed_tokens": dist.ColWiseParallel(
                         gather_output=True
                     ),
-                    "llama.position_embedding": dist.ColWiseParallel(),
-                    "llama.layers.*.self_attn.q_proj": dist.ColWiseParallel(
+                    f"{prefix}llama.position_embedding": dist.ColWiseParallel(),
+                    f"{prefix}llama.layers.*.self_attn.q_proj": dist.ColWiseParallel(
                         gather_output=True
                     ),
-                    "llama.layers.*.self_attn.k_proj": dist.ColWiseParallel(
+                    f"{prefix}llama.layers.*.self_attn.q_proj.lora_A": dist.ColWiseParallel(),
+                    f"{prefix}llama.layers.*.self_attn.k_proj": dist.ColWiseParallel(
                         gather_output=True
                     ),
-                    "llama.layers.*.self_attn.v_proj": dist.ColWiseParallel(
+                    f"{prefix}llama.layers.*.self_attn.k_proj.lora_A": dist.ColWiseParallel(),
+                    f"{prefix}llama.layers.*.self_attn.v_proj": dist.ColWiseParallel(
                         gather_output=True
                     ),
-                    "llama.layers.*.self_attn.o_proj": dist.RowWiseParallel(
+                    f"{prefix}llama.layers.*.self_attn.v_proj.lora_A": dist.ColWiseParallel(),
+                    f"{prefix}llama.layers.*.self_attn.o_proj": dist.RowWiseParallel(
                         is_input_parallel=False
                     ),
-                    "llama.layers.*.mlp.gate_proj": dist.ColWiseParallel(),
-                    "llama.layers.*.mlp.up_proj": dist.ColWiseParallel(),
-                    "llama.layers.*.mlp.down_proj": dist.RowWiseParallel(),
-                    "lm_head.weight": dist.ColWiseParallel(),
+                    f"{prefix}llama.layers.*.self_attn.o_proj.lora_B": dist.RowWiseParallel(),
```

Comment:
shall we config `f"{prefix}llama.layers.*.self_attn.o_proj.lora_A": dist.ColWiseParallel(),` ?